x-pack/plugin/transform/src/test/java/org/elasticsearch/xpack/transform/transforms/scheduling/TransformSchedulerTests.java

```diff
@@ -394,9 +436,27 @@ public void testRegisterMultipleTransforms() {
         String transformId2 = "test-register-transforms-2";
         String transformId3 = "test-register-transforms-3";
         TimeValue frequency = TimeValue.timeValueSeconds(5);
-        TransformTaskParams transformTaskParams1 = new TransformTaskParams(transformId1, TransformConfigVersion.CURRENT, frequency, false);
-        TransformTaskParams transformTaskParams2 = new TransformTaskParams(transformId2, TransformConfigVersion.CURRENT, frequency, false);
-        TransformTaskParams transformTaskParams3 = new TransformTaskParams(transformId3, TransformConfigVersion.CURRENT, frequency, false);
+        TransformTaskParams transformTaskParams1 = new TransformTaskParams(
```

Comment:
The changes in this file won't be needed if you leave the "shortcut" constructor without "unattended" parameter.